pkg/descheduler/controllers/migration/filter.go

```diff
@@ -121,6 +122,11 @@ func (r *Reconciler) Filter(pod *corev1.Pod) bool {
 	if !r.filterExistingPodMigrationJob(pod) {
 		return false
 	}
+
+	if !r.reservationFilter(pod) {
```

Comment:
pmj reconcile also needs to check if Reservation is used, but pod.spec.schedulerName is not in the list range, should it also be rejected?

Reply:
maybe this should be handled in a webhook, what do you think?